src/test/java/com/snowflake/kafka/connector/streaming/iceberg/IcebergIngestionSchemaEvolutionIT.java

```diff
@@ -261,7 +256,7 @@ public void testComplexRecordEvolution() throws Exception {
           new DescribeTableRow("APPROVAL", "BOOLEAN"),
           new DescribeTableRow("ARRAY1", "ARRAY(NUMBER(19,0))"),
           new DescribeTableRow("ARRAY2", "ARRAY(VARCHAR(16777216))"),
-          new DescribeTableRow("ARRAY3", "ARRAY(BOOLEAN)"),
+          new DescribeTableRow("ARRAY3", "ARRAY(VARCHAR(16777216))"),
```

Comment:
Sry I missed that somehow.

Reply:
No problem, it happens without the CI.

Reply:
funny thing, I didn't notice it in my PR as this is the expected result. Now this breaks my PR, have to rollback this line :D

Reply:
now i feel sad that i find such things "funny"